umap/static/umap/js/umap.js

```diff
@@ -258,10 +254,13 @@ L.U.Map.include({
       if (this.options.onLoadPanel === 'databrowser') this.openBrowser()
       else if (this.options.onLoadPanel === 'caption') this.displayCaption()
       else if (this.options.onLoadPanel === 'datafilters') this.openFilter()
+      else if (this.options.onLoadPanel === 'share') this.renderShareBox()
```

Comment:
Do we want to use `?share` instead, to be consistent with `?edit` and `?download` ?